### DISCUSSION TITLE
Release Notes CovPass(Check) Version 1.28

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#223] [Announcements]
# Release Notes CovPass(Check) Version 1.28

Release 1.28 is currently in final review and is expected to be released within the next couple of hours. 
  
### **Main Features CovPass:**  💉 

- DCC Import: DCCs can now be imported from the smartphone's gallery or file explorer (PDF only) into CovPass. DCCs can also be imported into CovPass using the sharing dialog of other apps. Before the import, the DCCs are checked for validity and lock entries. A pre-selection allows users to decide on the final DCCs to be imported. The documents can include multiple DCCs per picture/ PDF-file, however for the best results one QR-Code per page/ photo is recommended.  

- Update Certlogic: he rule engine CertLogic, which performs rule checking within CovPass(Check), has been updated. This now allows citiziens with incomplete birth dates to perform rule checks with their DCCs.

-----------------------------------------------------------------------------------------------------------

### **Main Features CovPassCheck:**  💉 

- Update Certlogic: he rule engine CertLogic, which performs rule checking within CovPass(Check), has been updated. This now allows citiziens with incomplete birth dates to perform rule checks with their DCCs.

-----------------------------------------------------------------------------------------------------------

An outlook for upcoming topics will be published soon!

-----------------------------------------------------------------------------------------------------------

**Thank you all for you support and feedback! Stay safe everyone** 👍 🎊 

Team CovPass and CovPassCheck! 🧑‍⚕️ 👩‍⚕️

**Comment by MikeMcC399** [comment]
@alexcimander 
Typos?:
"Release 1.26 is currently in final review" 1.28?
"he rule engine CertLogic" the rule ...?
